client/src/components/GlobalStyles.js

```diff
@@ -11,11 +13,21 @@ function GlobalStyles() {
         *:after {
```

Comment:
is it wanted that you only have `*:after` here and not `*` and `*:before`?

Reply:
no, and I corrected it